Makefile

```diff
@@ -104,7 +100,7 @@ pylint: compose-build
 	@$(RUN_NO_DEPS) pylint src/
 
 pytest: compose-build
-	@export FIDESCTL_TEST_MODE=True; docker-compose up -d $(IMAGE_NAME)
+	docker-compose up -e FIDESCTL_TEST_MODE=True -d $(IMAGE_NAME)
```

Comment:
I didn't think this worked when I was testing - https://github.com/docker/compose/issues/6177

Reply:
yeah i had to fix this too lol, pushed without testing 🙃